geotrek/feedback/views.py

```diff
@@ -38,7 +38,7 @@ class ReportList(CustomColumnsMixin, mapentity_views.MapEntityList):
     )
     model = feedback_models.Report
     filterform = ReportFilterSet
-    mandatory_columns = ['id', 'email', 'activity']
+    mandatory_columns = ['id', 'id', 'activity']
```

Comment:
I f you use id two times, you duplicate column. you should not use ID as label, use a  label property with custom _display